hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/SendContainerRequestHandler.java

```diff
@@ -130,7 +119,7 @@ public void onError(Throwable t) {
       responseObserver.onError(t);
     } finally {
       if (volume != null) {
-        volume.incCommittedBytes(-importer.getDefaultContainerSize() * 2);
+        volume.incCommittedBytes(-HddsServerUtil.requiredReplicationSpace(importer.getDefaultContainerSize()));
```

Comment:
Can we add one function in ContainerImporter which will return the value of  HddsServerUtil.requiredReplicationSpace(importer.getDefaultContainerSize()), and call this function where it's needed?